src/UserCom.Client/Model/Users/Requests/AddEventRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace UserCom.Model.Users.Requests
+{
+    public class AddEventRequest
+    {
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("timestamp")]
+        public long Timestamp { get; set; }
+
+        [JsonProperty("data")]
+        public object MemberDataJson { get; set; }
```

Comment:
I would change the name to Data as it can be any kind of data in the event, not just member specific data